device-types/FS/S5860-20SQ.yaml

```diff
@@ -0,0 +1,73 @@
+---
+manufacturer: FS
+model: S5860-20SQ
+slug: s5860-20sq
+part_number: s5860-20sq
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
```

Comment:
Please add a comment with a link to the datasheet in markdown format if you can find a publicly available one from the manufacturer
I believe [this is it](https://www.fs.com/products_support/preview.html?categories_id=3258&files_id=1263&model_name=S5860-48SC&is_datasheet=1)

Reply:
Fixed